docs/development.rst

```diff
@@ -94,15 +94,15 @@ version:
 
 .. _semantic versioning: http://semver.org/
 
-.. _release-1.0.0:
+.. _roadmap-release-1.0.0:
```

Comment:
These may or may not be linked externally. If there isn't a strong reason for this change, it might be best to leave the refs as they are. Probably not a huge loss otherwise.

Reply:
I got the feeling that it was too magical when I noticed it was overlapping with the changelog: https://github.com/readthedocs/sphinx_rtd_theme/blame/master/docs/changelog.rst#L60

So we _could_ remove the roadmap entry and "move" it to the roadmap, but that doesn't guarantee direct anchor links to not break.

I think we should just be explicit that this is a roadmap label vs a changelog label.

Reply:
Yeah, don't worry too much here. I can't recall what would have linked here, but at worst it's probably some old blog posts.

Reply:
The blog now gives build errors when that happens :)